sdk-api-src/content/d3d12shader/ns-d3d12shader-d3d12_shader_input_bind_desc.md

```diff
@@ -88,6 +88,7 @@ A <a href="/windows/desktop/api/d3dcommon/ne-d3dcommon-d3d_srv_dimension">D3D_SR
 
 The number of samples for a multisampled texture; when a texture isn't multisampled, the value is set to -1 (0xFFFFFFFF).
             This is zero if the shader resource is not a recognized texture.
+            If the resource is a structured buffer the field returns the stride of the struct.
```

Comment:
I don't think there's need to truncate words in documentation, especially to shorten one from 9 to 6 characters. In the context of structured buffers, the word commonly used in Microsoft documentation is "structure". `struct` is an HLSL keyword for a language concept, and as far as I'm aware, you can also declare a `StructuredBuffer<float4>` or other kind of a structured buffer with elements of numeric types, which will not involve an HLSL `struct`. The `D3D12_BUFFER_SRV` documentation says "size of each element in the buffer structure", though this wording is pretty confusing too (is the structure an array element, or actually the array itself?)